### DISCUSSION TITLE
Unable to open the downloaded PDF in mobile app

### DISCUSSION THREAD

**Discussion by nidhikachhawa24** [Sunbird-Ed/Community#595] [Bugs] [answered]
Hello all,

I'm using 4.6 version of mobile app.

**Steps:**
1. I've joined a course.
2. Opened the module in that course.
3. Clicked on download.
4. After the download is completed, getting option - **Open with**
5. When I click on **open with** icon, it is showing me this option 
![image](https://github.com/Sunbird-Ed/Community/assets/107101132/92aa0603-0056-467e-8f18-cfb56b1065ca)
6. When I click on **Google Bolo**, it opens play store and tells app not found.
7. When I try to search the downloaded content on my mobile local storage, not able to find it.

**Can anyone help me how this open with can be removed and this PDF can be find in local storage?**

@swayangjit

**Comment by swayangjit** [comment]
@nidhikachhawa24 The "OpenWith" is a different functionality that enables third-party vendors to launch the Sunbird content. 

Q1. Can anyone help me how this open with can be removed?
A. You can remove this from the UI.

Q2. Where to find the pdf ?
A. We are storing the content assets inside the app's private package. It will be like following
   `"/storage/emulated/0/Android/data/<package_name>/files/content/<content_identifier>/"`

**Reply by nidhikachhawa24** [reply]
@swayangjit 
I'm able to do both 1 and 2

But as the users are not much tech friendly, how can we directly show the downloaded files in the download folder?

**Reply by swayangjit** [reply, marked as answer]
@nidhikachhawa24 The purpose of downloading the content is to play the content via the content player offline. The users shouldn't access the files via FileManager, they can access them via content-player.

**Reply by nidhikachhawa24** [reply]
@swayangjit 
Thanks
